### PR TITLE
Add numerrin

### PR DESCRIPTION
**Pull request by nathanfranklin**
This PR adds numerrin to simphony-framework (and replaces the numerrin part of #36)

### CODE REVIEW

**Comment by nathanfranklin** [comment]
@itziakos ,  this is ready for review.  

fyi, i opened the following issues at simphony-numerrin related to integration:
* simphony/simphony-numerrin#5 
* simphony/simphony-numerrin#6

**Comment by itziakos** [comment]
@nathanfranklin, please merge master into this branch

**Comment by itziakos** [comment]
> @nathanfranklin, please merge master into this branch

done

File: Makefile [hunk 1]
```diff
@@ -119,6 +139,14 @@ simphony-mayavi:
 	@echo
 	@echo "Simphony Mayavi plugin installed"
 
+simphony-numerrin:
+	rm -Rf src/simphony-numerrin
+	git clone --branch 0.1.0 https://github.com/simphony/simphony-numerrin.git src/simphony-numerrin
+	cp src/simphony-numerrin/numerrin-interface/numerrin.so $(SIMPHONYENV)/lib/python2.7/site-packages/
+	(cd src/simphony-numerrin; python setup.py develop)
```
Comment (itziakos) [inline Makefile]: this should be probably `install` instead of `develop`

**Comment by nathanfranklin** [comment]
@itziakos , can you take one more look at this.

**Comment by itziakos** [comment]
:+1:

**Pull request merged**